features/product/managing_product_variants/adding_product_variant.feature

```diff
@@ -17,3 +18,13 @@ Feature: Adding a new product variant
         And I add it
         Then I should be notified that it has been successfully created
         And the "VODKA_WYBOROWA_PREMIUM" variant of the "Wyborowa Vodka" product should appear in the shop
```

Comment:
I should not be able to create variant of product with options without specifying option value.